repositories.yml

```diff
@@ -3,49 +3,173 @@ section:
+          or init container to ensure your application has a valid Conjur access token.
+        certification: "certified"
+        version: v0.15.0
+      - name: cyberark/secrets-provider-for-k8s
+        url: https://github.com/cyberark/secrets-provider-for-k8s
+        tool: Kubernetes
+        description: The Conjur Secrets Provider for K8s is deployed as an init
+          container in your application pod. It injects secrets from Conjur
+          into Kubernetes secrets, which are accessible to your application pod.
+        certification: "certified"
+        version: v0.2.0
+      - name: cyberark/conjur-service-broker
+        url: https://github.com/cyberark/conjur-service-broker
+        tool: Cloud Foundry
+        description: The Conjur service broker ensures your Cloud Foundry-deployed
+          applications are bootstrapped with a Conjur machine identity on deploy.
+        certification: "certified"
+        version: v1.0.0
+      - name: cyberark/cloudfoundry-conjur-buildpack
+        url: https://github.com/cyberark/cloudfoundry-conjur-buildpack
+        tool: Cloud Foundry
+        description: The Conjur buildpack brings the benefit of Summon to
+          your Cloud Foundry-deployed applications. Leverage your app's Conjur identity
+          to automatically inject the secrets your app needs into its environment
+          at runtime.
+        certification: "certified"
+        version: v2.1.1
+
+  - name: DevOps Tools
+    description: Conjur OSS integrations with DevOps tools.
+    repos:
+      - name: cyberark/ansible-conjur-host-identity
+        url: https://github.com/cyberark/ansible-conjur-host-identity
+        tool: Ansible
+        description: Ansible role to provide Conjur machine identity to application
+          hosts and install the Summon tool, which enables hosts to securely retrieve
+          credentials.
+        certification: "trusted"
+        version: v0.3.0
+      - name: cyberark/conjur-credentials-plugin
+        url: https://github.com/cyberark/conjur-credentials-plugin
+        tool: Jenkins
+        description: Conjur plugin for securely providing credentials to Jenkins jobs.
+        certification: "community"
+        version: v0.7
+      - name: cyberark/conjur-puppet
+        url: https://github.com/cyberark/conjur-puppet
+        tool: Puppet
+        description: Puppet module that simplifies the operation of establishing
+          Conjur host identity and allows authorized Puppet nodes to fetch secrets
+          from Conjur.
+        certification: "trusted"
+        version: v2.0.1
+      - name: cyberark/terraform-provider-conjur
+        url: https://github.com/cyberark/terraform-provider-conjur
+        tool: Terraform
+        description: Terraform provider that makes secrets in Conjur available in
+          Terraform manifests.
+        certification: "certified"
+        version: v0.1.0
+        
+  - name: Secretless Broker
+    description:
```

Comment:
Intentionally left out?

Reply:
yeah, we don't provide this in the landing page which is where I copied it from. there is only one repo in this section - the alternative could be that we have an "Application Access Delivery" section or something that includes both summon and secretless and is described as
> Tools to ensure your applications have access to the privileged systems they need.

but is this used for anything? can we leave it blank for now?

Reply:
> is this used for anything? can we leave it blank for now?

I think we should have something there even if it's inaccurate - or - make all descriptions optional and we remove this line.

Reply:
are descriptions required now? where are they used?

Reply:
Currently all descriptions are required in the parsing code so they are mandatory: https://github.com/cyberark/conjur-oss-suite-release/blob/master/pkg/repositories/repositories.go#L13